src/app/Scenes/Articles/Articles.tests.tsx

```diff
@@ -1,66 +1,41 @@
-import { flushPromiseQueue } from "app/utils/tests/flushPromiseQueue"
-import { renderWithWrappers } from "app/utils/tests/renderWithWrappers"
-import { RelayEnvironmentProvider } from "react-relay"
-import { createMockEnvironment, MockPayloadGenerator } from "relay-test-utils"
+import { screen, waitForElementToBeRemoved } from "@testing-library/react-native"
+import { setupTestWrapper } from "app/utils/tests/setupTestWrapper"
 import { ArticlesScreen } from "./Articles"
 
 describe("Articles", () => {
-  let mockEnvironment: ReturnType<typeof createMockEnvironment>
-  const TestRenderer = () => (
-    <RelayEnvironmentProvider environment={mockEnvironment}>
-      <ArticlesScreen />
-    </RelayEnvironmentProvider>
-  )
-
-  beforeEach(() => {
-    mockEnvironment = createMockEnvironment()
+  const { renderWithRelay } = setupTestWrapper({
+    Component: (props: any) => <ArticlesScreen {...props} />,
```

Comment:
Okay, just realized another thing, we don't need to pass props here since the [component](https://github.com/artsy/eigen/blob/main/src/app/Scenes/Articles/Articles.tsx#L48) that we are using doesn't have any 🎉 so we can change this line to:
```suggestion
    Component: () => <ArticlesScreen />,
```

Reply:
oh yeah true, since we're not using a query 😅 Thank you!!